### DISCUSSION TITLE
When can emby format Metadata be supported?

### DISCUSSION THREAD

**Discussion by xmumeyu** [sct/overseerr#2313] [Feature Requests]
I have two media servers EMBY and plex,

Plex installed XBMCnfoTVImporter plug-in to support emby Metadata

overseerr When reading emby format metadata, always prompt
[error][Plex Scan]: Failed to process Plex media {"errorMessage":"Unable to find TMDb ID","title":"Better Call Saul"}

When can you support it? thank you very much

**Comment by TheCatLady** [comment]
I don't see this happening, as it isn't an official Plex metadata agent... and Plex plugins are no longer officially supported.  I suggest that you use one of the supported Plex metadata agents.

That said, we do plan to support Emby servers in the future, in which case there would be additional scan jobs to detect availability from your Emby server.  There is no ETA for this, but you can subscribe to #295 for updates.
